homework-submission/main.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+function AddingForm({ addTodoItem }) {
+  return (
+    <div className='form'>
+      <form onSubmit={addTodoItem} className='Add'>
+        <input type='text' name='description' placeholder='Description' className='AddingList' />
+        <input type='date' name='deadline' placeholder='Deadline' />
+        <input type='submit' name='add' value='Add' className='submit' />
+      </form>
+    </div>
+  );
+}
+
+class TodoList extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      dynamicList: [
+        {
+          id: 1,
+          description: 'Get out of bed',
+          deadline: '2017-09-11',
+          done: true,
+        },
+        {
+          id: 2,
+          description: 'Brush teeth',
+          deadline: '2017-09-10',
+          done: false,
+        },
+        {
+          id: 3,
+          description: 'Eat breakfast',
+          deadline: '2017-09-09',
+          done: false,
+        },
+      ],
+    };
+  }
+
+  addItem = element => {
+    element.preventDefault();
+    const newTodo = {
+      id: this.state.dynamicList.length + 1,
+      description: element.target.description.value,
+      deadline: element.target.deadline.value,
+      done: false,
+    };
+    console.log(newTodo);
```

Comment:
No logs in your current or future projects, It would be a lot cleaner when you remove them after you are done.